src/components/authorPageComponents/Gallery/Gallery.js

```diff
@@ -1,13 +1,25 @@
 import React from 'react'
 
-import style from './Gallery.module.scss';
+import GalleryImage from 'react-grid-gallery';
+import galleryStyles from './Gallery.module.scss';
```

Comment:
What about bootstrap Carousel like this: [https://react-bootstrap.github.io/components/carousel/](https://react-bootstrap.github.io/components/carousel/) ? Of course, this is not necessary. And I can try to rewrite later.

Reply:
Bootstrap Carousel is good thing, but it increases the pictures too much and they become poor quality

Reply:
Yep, it can be a problem. Leave it as is.